invenio_accounts/sessions.py

```diff
@@ -152,3 +154,14 @@ def default_session_store_factory(app):
             accounts_session_redis_url))
     from simplekv.memory import DictStore
     return DictStore()
+
+
+class KVSessionInterfaceWithAnonymousSessions(KVSessionInterface):
```

Comment:
small impact: IMO the name indicates that we store sessions for anonymous. I would suggest `KVSessionInterfaceStoreAuthenticated`